trestle/core/commands/split.py

```diff
@@ -58,48 +86,92 @@ def _run(self, args: argparse.Namespace) -> int:
         logger.debug('Entering trestle split.')
         # get the Model
         args_raw = args.__dict__
-        if args_raw[const.ARG_FILE] is None:
-            logger.error(f'Argument "-{const.ARG_FILE_SHORT}" is required')
-            return 1
 
-        file_path = pathlib.Path(args_raw[const.ARG_FILE]).resolve()
-        if not file_path.exists():
-            logger.error(f'File {file_path} does not exist.')
-            return 1
-        content_type = FileContentType.to_content_type(file_path.suffix)
+        # remove any quotes passed in as on windows platforms
+        elements_clean: str = args_raw[const.ARG_ELEMENT].strip("'")
 
-        # find the base directory of the file
-        file_absolute_path = pathlib.Path(file_path)
-        base_dir = file_absolute_path.parent
+        file_path_list: List[Tuple[str, str]] = []
 
-        model_type, _ = fs.get_stripped_contextual_model(file_absolute_path)
+        if const.ARG_FILE in args_raw and args_raw[const.ARG_FILE] is not None:
+            file_path_list.append((args_raw[const.ARG_FILE], elements_clean))
+        else:
+            # cwd must be in the model directory if file to split is not specified
+            # find top directory for this model based on trestle root and cwd
+            model_dir = fs.get_project_model_path(pathlib.Path.cwd())
```

Comment:
Added issue: https://github.com/IBM/compliance-trestle/issues/641

Would like feedback on that issue.

Reply:
Added comments to #641 elaborating on the options and consequences.